try {
    try_statements
}
catch (exception_var_2){
    catch_statements_2
}
finally {
    finally_statements
}